server/src/main/java/io/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -85,6 +85,7 @@
   private static final Logger log = new Logger(JettyServerModule.class);
 
   private static final AtomicInteger activeConnections = new AtomicInteger();
+  private static final String GRACEFUL_SHUTDOWN_TIME = "graceful_shutdown_time";
```

Comment:
minor: `gracefulShutdownTimeout` would be more consistent with the property name.